packages/net/_test.pony

```diff
@@ -314,6 +314,9 @@ class _TestTCPWritevNotifyServer is TCPConnectionNotify
     end
     true
 
+  fun ref connect_failed(conn: TCPConnection ref) =>
+    None
```

Comment:
I am not sure now, if this good idea to ignore errors in tests :). Let me know what do you think about it